.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@ repos:
     hooks:
       - id: flake8
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.720  # Must match ci/requirements/*.yml
+    rev: v0.730  # Must match ci/requirements/*.yml
```

Comment:
IIUC `0.730` breaks some stuff so we are waiting for the next release

Reply:
I'll try to match master.